supported-targets.md

```diff
@@ -28,12 +33,17 @@
 | GigaDevice        | GD32F2      | ✔ Full        | v1.10.0            | stm32f1.c      | -
 | GigaDevice        | GD32F3      | ✔ Full        | v1.8.0             | stm32f1.c      | -
 | GigaDevice        | GD32F4      | ✔ Full        | v1.10.0            | stm32f4.c      | -
+| GigaDevice        | GD32F405    | ✔ Full        | v2.0.0             | stm32f.c       | -
 | GigaDevice        | GD32E230    | ✔ Full        | v1.8.0             | stm32f1.c      | -
 | GigaDevice        | GD32E5      | ✔ Full        | v1.10.0            | stm32f1.c      | -
 | ArteryTek         | AT32F40     | ✔ Full        | v1.9.0             | stm32f1.c      | -
 | ArteryTek         | AT32F405    | ✔ Full        | v2.0.0             | at32f43x.c     | -
 | ArteryTek         | AT32F41     | ✔ Full        | v1.9.0             | stm32f1.c      | -
+| ArteryTek         | AT32F403    | ✔ Full        | V2.0.0             | stm32f1.c      | -
+| ArteryTex         | AT32F413    | ✔ Full        | V2.0.0             | stm32f1.c      | -
+| ArteryTex         | AT32F413    | ✔ Full        | V2.0.0             | stm32f1.c      | -
```

Comment:
The V numbers here should be lower case please. (`v2.0.0`, not `V2.0.0`)

Reply:
Duplicate lines for ArteryTe**x** AT32F413, did you mean to add AT32F415 or AT32F421?